api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptDao.java

```diff
@@ -0,0 +1,30 @@
+package org.pmiops.workbench.cdr.dao;
+import org.pmiops.workbench.cdr.model.Concept;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
+
+public interface ConceptDao extends CrudRepository<Concept, Long> {
```

Comment:
Are clients going to have any options regarding ordering of concepts?

Reply:
Yes.  Will be updating api in next few weeks for full functionality